client/src/assets/fonts.css

```diff
@@ -21,6 +21,13 @@
 
 @font-face {
 	font-family: 'Spectral';
-	src: url('/assets/fonts/Spectral-Italic.ttf');
+	src: url('/assets/fonts/Spectral-MediumItalic.ttf');
```

Comment:
Do we still need Spectral-Italic in the repo then? It's ok if we want to keep it for later